src/main/java/com/denizenscript/depenizen/bukkit/events/projectkorra/EntityBendingDeathScriptEvent.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.depenizen.bukkit.events.projectkorra;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.projectkorra.projectkorra.event.EntityBendingDeathEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class EntityBendingDeathScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // projectkorra player|entity dies|death|killed by <ability>
```

Comment:
This is a really awkward event format -- maybe do `projectkorra <entity> dies` with switch `by:<ability>`